### PR TITLE
[ENH] Added afni.FWHMx interface

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

File: nipype/interfaces/afni/preprocess.py [hunk 1]
```diff
@@ -2160,3 +2162,200 @@ def _list_outputs(self):
...
+
+        if name == 'acf':
+            if isinstance(value, tuple):
+                return trait_spec.argstr + ' %s %f' % value
+            elif isinstance(value, string_types):
+                return trait_spec.argstr + ' ' + value
+        return super(FWHMx, self)._format_arg(name, trait_spec, value)
+
+    def _run_interface(self, runtime):
+        if platform == 'darwin':
```
Comment (chrisfilo) [inline nipype/interfaces/afni/preprocess.py]: Is this an AFNI wide or FWHM specific issue?

Comment (oesteban) [inline nipype/interfaces/afni/preprocess.py]: I only experienced it with the FWHMx interface, I don't know if it's for some other. Definitely not all.

Comment (oesteban) [inline nipype/interfaces/afni/preprocess.py]: Looks like despike also has the problem. Maybe worth setting for all afni interfaces.

Comment (chrisfilo) [inline nipype/interfaces/afni/preprocess.py]: +1

**Comment by oesteban** [comment]
This is ready to go. I have checked that tests pass (seems like Travis is queueing today)

**Comment by chrisfilo** [comment]
Are you planning to send the OSX fix a sseparate PR?

**Comment by oesteban** [comment]
It is included in this one since it is a minor change (added here https://github.com/nipy/nipype/pull/1361/files#diff-e0d822d271f09df14e2fa7444c9a4f3dR119)

**Comment by chrisfilo** [comment]
Thx - I somehow missed it.

**Pull request merged**